modules/rte_mis_state/rte_mis_state.module

```diff
@@ -279,6 +309,16 @@ function rte_mis_state_roles_ajax_callback(&$form, $form_state) {
   $current_user = \Drupal::currentUser();
   $user = User::load($current_user->id());
   $curr_user_role = $user->getRoles();
+  // Get the rte_mis_core settings.
+  $configSettings = \Drupal::configFactory()->get('rte_mis_core.settings');
+  // Get the categorization setting from config.
+  $categorizationDepth = $configSettings->get('location_schema.depth');
+  $termStorage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
+  $locationParentTerms = $termStorage->loadTree('location_schema', 0, $categorizationDepth, TRUE);
```

Comment:
`$termStorage->loadTree('location_schema', 0, $categorizationDepth, TRUE); ` set te lat parameter to FALSE, as we don't need to full term.

Reply:
ok